aurora/src/test/java/com/market/banica/aurora/config/JMXConfigTest.java

```diff
@@ -0,0 +1,178 @@
+package com.market.banica.aurora.config;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.market.banica.aurora.model.ChannelProperty;
+import com.market.banica.common.util.ApplicationDirectoryUtil;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.mockito.Spy;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.test.util.ReflectionTestUtils;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.file.Paths;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@ExtendWith(MockitoExtension.class)
+class JMXConfigTest {
+
+    private static final String CHANNEL_PREFIX = "testing-prefix";
+
+    private static final String INVALID_CHANNEL_PREFIX = "non-existent-prefix";
+
+    private static final String HOST = "localhost";
+
+    private static final String PORT = "1010";
+
+    private static final String EDITED_HOST = "edited-localhost";
+
+    private static final String TEST_FILE_NAME = "test";
+
+    @Mock
+    private ChannelManager channels;
+
+    @Spy
+    private final Map<String, ChannelProperty> channelPropertyMap = new HashMap<>();
```

Comment:
why is this @spy and initialized at the same time, I think @Spy needs to be removed